src/effects/EqualizationUI.cpp

```diff
@@ -38,6 +38,56 @@ BEGIN_EVENT_TABLE(EqualizationUI, wxEvtHandler)
 
 END_EVENT_TABLE()
 
+class EqualizationUIEditor : public EffectEditor, protected wxEvtHandler
```

Comment:
Might be in anonymous namespace

Reply:
oops

Reply:
Citing `DefaultEffectEditor` - `Inherit wxEvtHandler so that Un-Bind()-ing is automatic in the destructor`. I haven't tested that claim, I want minimal possible changes considering that we (possibly) want to target 3.3.3.

Reply:
I think it's unnecessary, because Bind() wasn't used, but let it stand if it is urgent and you have any doubt.